### ISSUE TITLE
Issue366 - BUG - Paint Options While Viewing Overlay Maps

### ISSUE THREAD

**Issue by tomtinkerer** [LESEM-PEWI/pewi3.0#366]
Open "Sandbox", go to Physical Feature maps, select any of the maps. Start dropping in a few Land Uses with the regular "click/drag paint" option, either clicking one at a time or dragging around the map. Now switch to the "box paint" option, and try adding some Land Uses - you will notice PEWI stays on the original setting.

Now try the same thing, but BEFORE opening an overlay map, switch to the "box paint" option. Now open an overlay map, and attempt to paint - it will default back to the "click/drag paint" option. 

Basically, the "box paint" option is turned off while a overlay map is displaying. Please fix this - both options ("click/drag paint" and "box paint") should be available.

**Comment by tomtinkerer** [comment]
Same as #423
